features/zones/Index.feature

```diff
@@ -8,7 +8,7 @@ Feature: The create Zone flow works
       | environment-kubernetes-radio-button | [data-testid='environment-kubernetes-radio-button'] |
       | ingress-input-switch                | [data-testid='ingress-input-switch']                |
       | egress-input-switch                 | [data-testid='egress-input-switch']                 |
-    When I visit the "/zones/-create" URL
+    When I visit the "/zone-cps/-create" URL
```

Comment:
This one doesn't seem to be inside `zones`, what are we actually creating here a `zone-cp` or a `zone`? (also see below comment on route nesting that could be related)

Reply:
I think this is actually for _Zones_ rather than Zone CPs. I’m not totally sure, but I think that’s accurate. I’ve moved the URL to `/zones`.